package.json

```diff
@@ -22,12 +26,20 @@
     "test:prep": "lerna run test:prep",
     "lint": "npm --workspaces run lint",
     "lint:fix": "npm --workspaces run lint:fix",
+    "lint:staged": "lint-staged",
+    "lint:commit": "commitlint",
     "preversion": "npm run rebuild",
+    "prepare": "husky install",
```

Comment:
`prepare`  runs on publishing too. Why not `postinstall`?

sidenote: husky is fairly complex. Do we need more than https://www.npmjs.com/package/pre-commit ?

Reply:
sounds good. husky is also fine.

Let's address the script choice though.

Reply:
I think you may be right about `postinstall` if we want to avoid `husky install` on publish--but only because we're in a workspace root.  `prepare` should probably be calling `npm run rebuild` instead of `preversion`, then.